### PR TITLE
feat: add RTL support to roving tabindex

### PR DESCRIPTION
**Pull request by yuriy-fix**
Connected to #65 

Fixes vaadin/vaadin-tabs#105

### CODE REVIEW

File: vaadin-list-mixin.html [hunk 1]
```diff
@@ -169,16 +169,23 @@
       const condition = item => !(item.disabled || this._isItemHidden(item));
       let idx, increment;
 
+      const hostIsRTL = this.getAttribute('dir') === 'rtl';
```
Comment (web-padawan) [inline vaadin-list-mixin.html]: Consider using the same code as in vaadin/component-mixins#33

```js
const isRTL = !this._vertical && this.getAttribute('dir') === 'rtl';
const dirIncrement = isRTL ? -1 : 1;
```

File: test/vaadin-list-mixin.html [hunk 2]
```diff
@@ -281,6 +281,47 @@
...
+            expect(list.items[1].focused).to.be.true;
+          });
+
+          it('should move focus to prev element on "arrow-up" keydown', () => {
+            arrowDown(list);
+            arrowUp(list);
+            expect(list.items[0].focused).to.be.true;
+          });
+
+          it('should move focus to prev element on "arrow-left" keydown', () => {
```
Comment (web-padawan) [inline test/vaadin-list-mixin.html]: Please change the test suite description, should be `next element`.

File: test/vaadin-list-mixin.html [hunk 3]
```diff
@@ -281,6 +281,47 @@
...
+            list.orientation = 'horizontal';
+            arrowLeft(list);
+            expect(list.items[1].focused).to.be.true;
+          });
+
+          it('should move focus to next element on "arrow-right" keydown', () => {
```
Comment (web-padawan) [inline test/vaadin-list-mixin.html]: Please change the test suite description, should be `prev element`.

File: test/vaadin-list-mixin.html [hunk 4]
```diff
@@ -281,6 +281,47 @@
...
+            list.orientation = 'horizontal';
+            arrowLeft(list);
+            arrowRight(list);
+            expect(list.items[0].focused).to.be.true;
+          });
+
+          it('should move focus to last element on "home" keydown', () => {
+            list._focus(3);
```
Comment (web-padawan) [inline test/vaadin-list-mixin.html]: This line should be moved from `home` test to `end` test below (to ensure End moves to first item).

File: vaadin-list-mixin.html (hunk 1 above)
Comment (yuriy-fix) [reply vaadin-list-mixin.html]: Done.

File: test/vaadin-list-mixin.html (hunk 2 above)
Comment (yuriy-fix) [reply test/vaadin-list-mixin.html]: `next` element was here in the meaning of the `visually previous` element (not the DOM order as it is visual tests).
But if it easier this way, updated.

File: test/vaadin-list-mixin.html (hunk 3 above)
Comment (yuriy-fix) [reply test/vaadin-list-mixin.html]: `next` element was here in the meaning of the `visually previous` element (not the DOM order it is visual tests).
But if it easier this way, updated.

File: test/vaadin-list-mixin.html (hunk 4 above)
Comment (yuriy-fix) [reply test/vaadin-list-mixin.html]: Good catch! Updated.

**Pull request merged**